assets/section-multicolumn.css

```diff
@@ -223,12 +223,31 @@
 }
 
 .multicolumn-card {
+  position: relative;
+  border-radius: var(--text-boxes-radius);
   border: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
+}
+
+.multicolumn-card > .multicolumn-card__image-wrapper--full-width:not(.multicolumn-card-spacing) {
+  border-top-left-radius: calc(var(--text-boxes-radius) - var(--text-boxes-border-width));
+  border-top-right-radius: calc(var(--text-boxes-radius) - var(--text-boxes-border-width));
+  overflow: hidden;
+}
+
+.multicolumn-card:after {
+  content: '';
+  position: absolute;
   border-radius: var(--text-boxes-radius);
-  box-shadow: var(--text-boxes-shadow-horizontal-offset)
-    var(--text-boxes-shadow-vertical-offset)
-    var(--text-boxes-shadow-blur-radius)
-    rgba(var(--color-foreground), var(--text-boxes-shadow-opacity));
+  box-shadow: var(--text-boxes-shadow-horizontal-offset) var(--text-boxes-shadow-vertical-offset) var(--text-boxes-shadow-blur-radius) rgba(var(--color-foreground), var(--text-boxes-shadow-opacity));
+  width: calc(var(--text-boxes-border-width) * 2 + 100%);
+  height: calc(var(--text-boxes-border-width) * 2 + 100%);
+  top: calc(var(--text-boxes-border-width) * -1);
+  left: calc(var(--text-boxes-border-width) * -1);
+  z-index: -1;
+}
+
```

Comment:
That's how I'm planning on dealing with shadows and overflow: hidden sliders. Basically adding some padding (equal to shadow offset) above or below the slides. I guess it's time to add it to the list. 

@melissaperreault I don't want to handle it in this PR because it isn't specific to multicolumn, but can we prioritize an issue and consider if that approach sounds reasonable?

Reply:
Issue here https://github.com/Shopify/dawn/issues/1027

Reply:
Thanks and good catch! Didn't see this! 🙏